tests/pybind11_tests.h

```diff
@@ -57,7 +57,9 @@ union IntFloat {
 class UnusualOpRef {
 public:
     using NonTrivialType = std::shared_ptr<int>; // Almost any non-trivial type will do.
-    NonTrivialType operator&() { return non_trivial_member; } // UNUSUAL operator.
+    // UNUSUAL operators:
```

Comment:
What about: "Overriding & operator should not break pybind11."

Reply:
Done (928b5b4750733ef4edc9bf4ce8839fe8a944ebe4). Thanks!